nwb_conversion_tools/nwbconverter.py

```diff
@@ -62,24 +68,33 @@ def get_metadata(self):
 
     def run_conversion(self, metadata_dict, nwbfile_path=None, save_to_file=True, stub_test=False):
         """Build nwbfile object, auto-populate with minimal values if missing."""
+
+        # Minimal values
         nwbfile_kwargs = dict(
-                session_description="no description",
-                identifier=str(uuid.uuid4()),
-                session_start_time=datetime.now()
+            session_description="no description",
+            identifier=str(uuid.uuid4()),
+            session_start_time=datetime.now()
+        )
+        # convert ISO 8601 string to datetime
+        if 'NWBFile' in metadata_dict and isinstance(metadata_dict['NWBFile']['session_start_time'], str):
+            metadata_dict['NWBFile']['session_start_time'] = datetime.fromisoformat(
+                metadata_dict['NWBFile']['session_start_time']
             )
 
-        if 'NWBFile' in metadata_dict:
-            nwbfile_kwargs.update(metadata_dict['NWBFile'])
-
-        if 'Subject' in metadata_dict:
-            nwbfile_kwargs.update(subject=Subject(**metadata_dict['Subject']))
-
+        # Initiate nwbfile
+        nwbfile_kwargs.update(metadata_dict['NWBFile'])
         nwbfile = NWBFile(**nwbfile_kwargs)
 
+        # add Subject
+        if 'Subject' not in metadata_dict:
+            metadata_dict['Subject'] = dict()
+        nwbfile.subject = Subject(**metadata_dict['Subject'])
```

Comment:
We shouldn't automatically create an empty Subject

Reply:
820547d